src/hermes/cli.py

```diff
@@ -159,3 +159,7 @@ def main(ctx: click.Context, *args, **kwargs) -> None:
 main.add_command(workflow.process)
 main.add_command(workflow.deposit)
 main.add_command(workflow.postprocess)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
**question:** Why did you add another main block here? Is there another reason that just convenience? Since we have

- the script entry point to run `hermes`
- the `__main__.py` to run `python -m hermes`

I don't see the point here to add yet another main routine.

Reply:
Oops, I completely missed the `__main__.py` 🤦🏻‍♂️ Reverted in 48c93e9adb39cf410ee16d9d975c8cdfe326b06c